Settings-default.toml

```diff
@@ -51,6 +51,12 @@ enable_buffering = false
 # Use this option if another folder should be used.
 #log_directory = "/var/log/"
 
+[ogc.default_time]
+# type "Value" with start/end as ISO strings or "Now" for using the current time, use wcs.default_time etc. for override
+type = "Value"
```

Comment:
Are Value and Now case sensitive? And if so, could we change that? Or at least also support value and now?

Reply:
I use serde type tag which per default uses the name of the enum variant as it is. I don't know if we want to apply the same guidelines like in the API. But then we would also need to use camelCase for `default_time`. I think in order to make it case insensitive we need a custom deserializer but I am not sure about that.

I would leave it like this but I can also change it to lower case if you want.

Reply:
Maybe just allow the lowercase variant as well and then we are fine. I guess chances are large that people write "value" but less likely that they write "vALue".

Reply:
#[serde(alias = "now")]